.github/workflows/test-e2e.yml

```diff
@@ -92,10 +98,18 @@ jobs:
         run: wget -qO - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | TAG=$K3D_VERSION bash
       - name: Provision SKR cluster
         run: |
-          k3d cluster create skr -p 10080:80@loadbalancer -p 10443:443@loadbalancer --k3s-arg '--disable=traefik@server:0'
+          k3d cluster create skr \
+            -p 10080:80@loadbalancer \
+            -p 10443:443@loadbalancer \
+            --k3s-arg '--disable=traefik@server:0'
       - name: Provision KCP cluster
         run: |
-          kyma provision k3d --name=kcp -p 9443:443@loadbalancer -p 9080:80@loadbalancer -p 9081:8080@loadbalancer --ci --registry-port 5111
+          k3d cluster create kcp \
+            -p 9443:443@loadbalancer \
+            -p 9080:80@loadbalancer \
+            -p 9081:8080@loadbalancer \
+            --registry-create k3d-kcp-registry:5111 \
+            --k3s-arg '--disable=traefik@server:0'
```

Comment:
In an older commit of this PR you used the newly introduced `K8S_Version` envVar. What happened to it? How it is used now? Is there some magic happening? Or how will the k8s version be set?


Before it was used with `--image rancher/k3s:v$K8S_VERSION-k3`

Reply:
🤦 Wow... during reformatting actually threw away the important change.
Thanks, PTAL again